src/Apps/Sell/Routes/__tests__/TitleRoute.jest.tsx

```diff
@@ -0,0 +1,76 @@
+import { screen } from "@testing-library/react"
+import { fireEvent, waitFor } from "@testing-library/react"
+import { TitleRoute } from "Apps/Sell/Routes/TitleRoute"
+import { setupTestWrapperTL } from "DevTools/setupTestWrapper"
+import { useRouter } from "System/Router/useRouter"
+import { graphql } from "react-relay"
+import { SubmissionRoute } from "Apps/Sell/Routes/SubmissionRoute"
+
+const mockUseRouter = useRouter as jest.Mock
+const mockPush = jest.fn()
+const mockReplace = jest.fn()
+
+jest.mock("System/Router/useRouter", () => ({
```

Comment:
Going to need to rebase

Reply:
Yep, saw your message, thanks for letting us know!